server/sonar-db-core/src/main/java/org/sonar/db/DatabaseUtils.java

```diff
@@ -236,6 +236,10 @@ public static void closeQuietly(@Nullable ResultSet rs) {
     }
   }
 
+  /**
+   * @deprecated replaced by Commons Land {@code StringUtils.repeat(sql, separator, count)
+   */
+  @Deprecated
   public static String repeatCondition(String sql, int count, String separator) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)